modules/ossm-migrating-workloads-using-the-istio-injection-label-with-cert-manager.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-injection-label-with-cert-manager_{context}"]
+= Migrating workloads using the Istio injection label with cert-manager
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+Revision tags are not used in this example for simplicity. When migrating large meshes, you can use revision tags to avoid re-labeling all namespaces during future 3.y.z upgrades.
```

Comment:
This part is taken from `Cluster-wide migration using the istio revision label` and should not be in `Cluster-wide migration using the istio injection label`.
I would suggest to just copy/paste whole `Migrating workloads by using the istio injection label` chapter from [here](https://89324--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.html#ossm-migrating-workloads-using-the-istio-injection-label_ossm-migrating-cluster-wide-assembly) so we are sure there are no other problems. It can be taken as is 1:1

Reply:
Nevermind. You're talking about migrating workloads here. I was thinking of the first procedure: migrating cluster-wide using instio injection with cert manager.

I'm with you -- we're just dissing the task for migrating the workloads.